src/static/css/content.less

```diff
@@ -316,6 +322,19 @@ figure {
 // Initiate loop
 .color-swatch-loop( length(@colors) );
 
+.color-table {
+    width: 100%;
+    margin-bottom: 40px;
+
+    .swatch_field {
+        width: 200px;
+        height: 2.5em;
```

Comment:
I prefer the white to the gray line, but can there be no line (or a line that's the same color as the swatch above it)? 

Separately, it would be great if we could adjust the spacing of the columns a bit. The names are pretty tight against the swatch compared to the spacing between Name->HEX or GRB->CMYK.